volume/volume_test.go

```diff
@@ -14,31 +29,67 @@ func TestNewVolumeEntry(t *testing.T) {
 }
 
 func TestNewVolumeEntryFromEmptyRequest(t *testing.T) {
+	req := new(VolCreateRequest)
+	v := NewVolumeEntry(req)
+	tests.Assert(t, v == nil)
+}
 
-	req := VolCreateRequest{}
+func find(haystack []string, needle string) bool {
 
-	v := NewVolumeEntry(req)
-	tests.Assert(t, len(v.Name) == 0)
+	for _, hay := range haystack {
+		if hay == needle {
+			return true
+		}
+	}
+
+	return false
 }
 
-func TestNewVolumeEntryFromRequest(t *testing.T) {
+func TestNewVolumeEntryFromRequestBricksRootPartition(t *testing.T) {
```

Comment:
This is testing newBrickEntries no NewVolumeEntry. The name for this test is incorrect.